.gitlab-ci.yml

```diff
@@ -74,7 +74,10 @@ stages:
   - source_test_junit_upload
   - binary_build
   - package_deps_build
-  - kernel_matrix_testing
+  - kernel_matrix_testing_prepare
+  - kernel_matrix_testing_system_probe
+  - kernel_matrix_testing_security_agent
+  - kernel_matrix_testing_cleanup
```

Comment:
Why did you add new stages?

Reply:
Mostly because it felt harder to find the jobs in the Gitlab pipelines page and understand the stages, exacerbated with the name cropping the interface does. It has no functional effect as all jobs are using `needs`, so I can revert it back quickly if you prefer.

Reply:
I prefer the separation. I think we can change the dashboards.